scripts/wcag-test.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * Uses pa11y-ci and pa11y configs to perform WCAG 2 AA accessibility tests
+ */
+
+const chalk = require('chalk');
+const { log, error } = console;
+const childProcess = require('child_process');
+const {
+  createCiPa11yConfigs,
+  createLocalPa11yConfigs,
+} = require('./update-pa11y-config.js');
+
+// use correct pa11y config based on args provided
+const ci = process.argv.some((arg) => arg === '--ci');
+const baseCommand = 'cd wcag && npx pa11y-ci';
+const command = ci
+  ? `${baseCommand} --config .pa11yci.ci.json`
+  : `${baseCommand} --config .pa11yci.local.json`;
+
+// update pa11y configs using data in vr backstop scenarios
+ci ? createCiPa11yConfigs() : createLocalPa11yConfigs();
```

Comment:
not sure how this passed lint - use ternery statements when you want to return a value. Just use a boring if statement here.